changelog/release-schedule.dd

```diff
@@ -0,0 +1,56 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(UL
+    $(LI New release are published every $(I two) months, on the first day of every uneven month.)
+    $(LI Two weeks before a new release `master` is merged into `stable` and a first beta is released.)
+    $(LI Patch releases are usually released on the first day of every even month.)
+    $(LI In the unfortunate case of serious regressions being found, more point releases or betas may be released.)
+)
+
+$(P The release schedule for 2018 is as follows:)
+
+    $(DIVC release-schedule,
+        $(TABLE
+            $(MINOR_RELEASE 2018-03-01, 2.079.0)
+            $(PATCH_RELEASE 2018-04-01, 2.079.1)
+            $(BETA_RELEASE 2018-04-15, 2.080.0)
+            $(MINOR_RELEASE 2018-05-01, 2.080.0)
```

Comment:
1st of every odd month and 15th of every even month is rather straighforward.
How about just giving a brief example with 4 dates for 2018 and linking to the calendar (#2222) instead?